clang/unittests/Format/ConfigParseTest.cpp

```diff
@@ -187,7 +187,6 @@ TEST(ConfigParseTest, ParsesConfigurationBools) {
   CHECK_PARSE_BOOL(RemoveSemicolon);
   CHECK_PARSE_BOOL(SkipMacroDefinitionBody);
   CHECK_PARSE_BOOL(SpacesInSquareBrackets);
-  CHECK_PARSE_BOOL(SpaceInEmptyBlock);
```

Comment:
Don't we still want this? I mean its still valid config

Reply:
We need a check, but this does not work anymore, since the member is gone.

Reply:
Yeah, it's same with [SpaceInEmptyParentheses](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#spaceinemptyparentheses), currently I checked only [here](https://github.com/llvm/llvm-project/pull/93634/files#diff-6b5b06a9a335ec5939905df81f1def72c3f869e24c758a8dd7dc1706873a77f0R643-R646).